example/image-classification/predict-cpp/README.md

```diff
@@ -1,13 +1,24 @@
 # Image Classification Example of C++
-This is a simple predictor which shows how to use c api for image classfication.
+This is a simple predictor which shows how to use the MXNet C++ API for image classification with a pre-trained ImageNet model.
```

Comment:
This example actually uses the C Predict API, and not the C++ API. They are not the same.